mne/preprocessing/ica.py

```diff
@@ -1198,7 +1220,14 @@ def find_bads_ecg(self, inst, ch_name=None, threshold=None, start=None,
 
         if method == 'ctps':
             if threshold is None:
+                warn('The default threshold will change from 0.25 to'
+                     'automatical computation in version 0.22.',
```

Comment:
```suggestion
                     'automatic computation in version 0.22.',
```
you should tell users set threshold to 'auto' to avoid the warning